Cargo.toml

```diff
@@ -7,8 +7,7 @@ license = "Apache-2.0"
 edition = "2018"
 
 [dependencies]
-ethereum-types = { version = "0.10.0", default-features = false }
-primitive-types = { version = "0.8", default-features = false, features = ["rlp"] }
```

Comment:
Does the new ethereum-types include primitive-types with feature rlp?

Reply:
It seems ethereum-types by default just includes primitive-types without rlp:
https://github.com/paritytech/parity-common/blob/master/ethereum-types/Cargo.toml#L14

Also, the direct dependency on primitive-types wasn't necessary as it we weren't importing `primitive_types` anywhere.

Anyway the code compiles and the tests pass so we should be in the clear:
```
   Compiling hex-literal v0.2.1
   Compiling ethash v0.5.0 (/home/vincent/code/ethash)
    Finished test [unoptimized + debuginfo] target(s) in 0.87s
     Running target/debug/deps/ethash-67870bdcf6197ef6

running 1 test
test tests::hashimoto_should_work ... test tests::hashimoto_should_work has been running for over 60 seconds
test tests::hashimoto_should_work ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 121.66s

   Doc-tests ethash

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```